readme.md

```diff
@@ -135,6 +135,9 @@ The backend of this project is built using [FastAPI](https://fastapi.tiangolo.co
 | `JWT_SECRET` | _CHANGEME_ | JWT-based auth secret key, used in the process of signing a token |
 | `WEBDOMAIN` | "http://localhost" | The base URL of the web application |
 | `MAGIC_LINK_CHAT_TEMPLATE` | _CHANGEME_ | A template for magic link message, e.g. "You can login into APP_NAME by clicking this link: {magic_link}" |
+| `GOOGLE_APPLICATION_CREDENTIALS_PATH` | _CHANGEME_ | Path to the service account JSON key file location used for authentication and accessing Google Cloud services (Development only) |
+| `GOOGLE_APPLICATION_CREDENTIALS` | _CHANGEME_ | JSON key file name used for authentication and accessing Google Cloud services |
+| `BUCKET_NAME` | _CHANGEME_ | Bucket name for a storage object (offered by Google Cloud) |
```

Comment:
Please also document `STORAGE_LOCATION`.

Reply:
For now I think we doesn't use the `STORAGE_LOCATION`, since we decide to store the media into google bucket. Do we need to add into the readme?  

The storage location is there for now because I take the storage code from our prev project. I'm thinking to remove that part of code if we will not use it.

Reply:
Yeah, remove it if we don't use it.